src/main/java/mod/chiselsandbits/core/api/ChiselAndBitsAPI.java

```diff
@@ -322,4 +330,22 @@ public void endUndoGroup(
 		UndoTracker.getInstance().endGroup( player );
 	}
 
+	@Override
+	@SideOnly( Side.CLIENT )
+	public KeyBinding getKeyBinding(
+			ModKeyBinding modKeyBinding )
+	{
+		switch ( modKeyBinding.type )
```

Comment:
This particular method screams out as being a likely cause of future issues, relying on the ordinals to line up, and to remain the properly synchronized over time will likely fail at some point.

I admit I don't have too many better ideas at the moment because we still need to keep the API separate from the core, but at minimum perhaps we could add a tests that verifies that this method returns an Enum with the correct name? and then add a simple test to the init phase to sanity check?

or perhaps some more thought is needed, its really the only sore spot from what I can see.